style.css

```diff
@@ -0,0 +1,100 @@
+html {
+  /* Setting a base font size of 10px give us easier rem calculations
+     Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+   */
+  font-size: 10px;
```

Comment:
Please remember that always remove the comments from your code, This is why to keep your code clean and this is best html and css practices. 👍